rails:
  both_sides: true        # spawn rails on both sides of aisle
  width: 0.5
  height: 0.065
  length: 50
  rail_width: 0.065

plants:
  both_sides: true        # spawn plants on both sides of aisle
  width: 0.5
  height: 1.5
  length: 50

world:
  name: greenhouse        # under this name world will be saved, default `greenhouse` if not specified
  number_of_rows: 11
  row_width: 1.5          # distance between rows
  aisle_width: 3.5
  aisle_origin_x: -2.75   # location of middle of aisle in the world. 
                          # For example if point (0,0) is 1 m inside row 1 then aisle_origin = -aisle_width/2 - 1


## add obstacles - currently supported are: box
# obstacles:
#   box1:
#     pos_x: 5.0
#     pos_y: 0.0
#   box2:
#     pos_x: -2.75
#     pos_y: -5.0